sdk/iot/hub/tests/cmocka/test_az_iot_hub_client_sas.c

```diff
@@ -30,9 +30,9 @@
 
 enable_precondition_check_tests()
 
-// Tests
+    // Tests
 
-static void az_iot_sas_token_get_document_NULL_document_fails(void** state)
+    static void az_iot_sas_token_get_document_NULL_document_fails(void** state)
```

Comment:
This and above doesn't look right.

Reply:
This is the same problem as described in https://github.com/Azure/azure-sdk-for-c/pull/583#discussion_r408581889.

The `enable_precondition_check_tests` #define causes this formatting issue.